presto-mongodb/src/test/java/com/facebook/presto/mongodb/MongoQueryRunner.java

```diff
@@ -68,7 +68,9 @@ public static MongoQueryRunner createMongoQueryRunner(Iterable<TpchTable<?>> tab
 
             Map<String, String> properties = ImmutableMap.of(
                     "mongodb.seeds", queryRunner.getAddress().getHostString() + ":" + queryRunner.getAddress().getPort(),
-                    "mongodb.socket-keep-alive", "true");
+                    "mongodb.socket-keep-alive", "true",
```

Comment:
Sorry for the back and forth, I'm not so familiar with Mongo, so I'm learning as I go.   Now that I have a better understanding of tagsets, I see why it can't be tested. I think it's okay to leave this out and not test tagsets.